### PR TITLE
fix(vsts): Fix script that determines which tests should be run

### PR DESCRIPTION
**Pull request by abhipsaMisra**

### CODE REVIEW

File: vsts/determine_tests_to_run.ps1 [hunk 1]
```diff
@@ -13,11 +13,6 @@ function IsPullRequestBuild
 
 function ShouldSkipIotHubTests 
 {	
-	if (IsPullRequestBuild)
```
Comment (vinagesh) [inline vsts/determine_tests_to_run.ps1:16]: Were we not running tests earlier?

Comment (vinagesh) [reply vsts/determine_tests_to_run.ps1:16]: Ahh draft PR, missed my eye :)

Comment (abhipsaMisra) [reply vsts/determine_tests_to_run.ps1:16]: we were actually doing the opposite, running tests irrespective of changes.

**Pull request merged**